Features/ManageWallets/Sources/Views/WalletListItemView.swift

```diff
@@ -7,6 +7,7 @@ import Components
 import Primitives
 import Localization
 import PrimitivesComponents
+import AvatarToolkit
```

Comment:
```suggestion
import AvatarToolkit
```

How about `WalletAvatar` ?